scripts/installer/postgres-setup

```diff
@@ -57,20 +57,23 @@ $pod_name = $ENV{'MY_POD_NAME'};
 if ($pod_name eq "start-glassfish")
 {
     print "Init container starting \n";
+
     $CONFIG_DEFAULTS{'POSTGRES_SERVER'} = $ENV{"POSTGRES_SERVER"} . "." .  $ENV{"POSTGRES_SERVICE_HOST"};
     $CONFIG_DEFAULTS{'POSTGRES_DATABASE'} = $ENV{"POSTGRES_DATABASE"};
     $CONFIG_DEFAULTS{'POSTGRES_USER'} = $ENV{"POSTGRES_USER"};
-    $CONFIG_DEFAULTS{'POSTGRES_ADMIN_PASSWORD'} = $ENV{"POSTGRES_ADMIN_PASSWORD"};
     $CONFIG_DEFAULTS{'POSTGRES_PASSWORD'} = $ENV{"POSTGRES_PASSWORD"};
+
```

Comment:
I am not sure why you are making all the changes in this section.

Reply:
This appears to be me being obsessive about code format. Not necessary, removed changes.